### ISSUE TITLE
Donation BuyMeACoffee

### ISSUE THREAD

**Issue by pquerner** [pmoreno-rodriguez/grav-theme-future2021#44]
Hey,

thanks for adding the donation button. I tried using that service, however there seem to be a issue with the payment and I always get an error.

For 1 time only, could you share your paypal.me link with me, so I can donate using paypal?

**Comment by pmoreno-rodriguez** [comment]
Hi @pquerner. I have made a test donation with the buymecoffee button and everything has worked correctly. I don't know if the problem is with your card. Anyway, I've created a link on Paypal, as you suggest, which I'll embed on my GitHub pages later.

However, the work I do for the Grav user community, with the design of themes and collaborations in forums and with some translations of the Gantry framework, I do it in a completely altruistic way, so you are not under any kind of obligation to make donations.

Here is the PayPal link, in case you still want to contribute. [https://paypal.me/birdydesigngrav?country.x=ES&locale.x=es_ES](https://paypal.me/birdydesigngrav?country.x=ES&locale.x=es_ES)

Thanks for everything.
